tests/acceptance/test_issue_tag_values.py

```diff
@@ -26,13 +29,12 @@ def create_issue(self):
         return self.store_event(data=event_data, project_id=self.project.id)
 
     def test_user_tag(self):
-        event = self.create_issue()
-        self.page.visit_tag_values(self.org.slug, event.group_id, "user")
-
-        assert self.browser.element_exists_by_test_id("group-tag-mail")
+        with self.feature("organizations:eventuser-from-snuba"):
```

Comment:
Seems like the writing to `EventUser` stops unconditionally in this PR, but for reading, we still rely on a feature flag being on? Could this be a problem for intstances (self-hosted, ST) that do not have the feature flag enabled?

Reply:
This is a good point, I think we should default `organizations:eventuser-from-snuba` to true in the config, just to avoid any problems in ST.